pkgs/development/python-modules/cryptography/default.nix

```
@@ -54,12 +54,12 @@ buildPythonPackage rec {
 
   nativeBuildInputs = lib.optionals (!isPyPy) [
     cffi
-    pkg-config
   ] ++ [
     rustPlatform.cargoSetupHook
     setuptools-rust
     cargo
     rustc
+    pkg-config
   ];
```

```suggestion
  nativeBuildInputs = [
    rustPlatform.cargoSetupHook
    setuptools-rust
    cargo
    rustc
    pkg-config  
  ] ++ lib.optionals (!isPyPy) [
    cffi
  ];
```

Would love if this were turned around.